python/cuml/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -16,26 +16,19 @@
 #
 
 
-# distutils: language = c++
-from cuml.internals.api_decorators import (
-    device_interop_preparation,
-    enable_device_interop,
-)
-from cuml.internals.safe_imports import (
-    cpu_only_import,
-    gpu_only_import,
-    gpu_only_import_from,
-    null_decorator,
-)
-
-np = cpu_only_import('numpy')
-nvtx_annotate = gpu_only_import_from("nvtx", "annotate", alt=null_decorator)
-rmm = gpu_only_import('rmm')
+import numpy as np
 
 import cuml.internals
+import cuml.internals.nvtx as nvtx
 from cuml.common import input_to_cuml_array
 from cuml.common.doc_utils import generate_docstring, insert_into_docstring
 from cuml.internals import logger
+
+# distutils: language = c++
```

Comment:
Is this comment still needed? What does it do? (Happy to defer to a later PR)

Reply:
These comments tell cython to use c++ instead of C for the generated sources (at the source level). Clearly since the comment got juggled around and things still build fine it's not needed (it's also not present in all our `.pyx` files).

I suspect some CI thing will fail here since I didn't run any locally. If that happens I'll drop this comment since it got moved around accidentally (I _think_ by cython lint sorting imports, but maybe by me :shrug:). We probably could drop them from all cython files later on.

Reply:
As expected, there were a few fixups needed for CI. I've moved the (useless but harmless) comment back to where it was. Can remove them all later.